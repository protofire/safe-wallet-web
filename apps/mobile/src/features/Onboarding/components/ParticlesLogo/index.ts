import { ParticlesLogo } from './ParticlesLogo'
export { ParticlesLogo }
