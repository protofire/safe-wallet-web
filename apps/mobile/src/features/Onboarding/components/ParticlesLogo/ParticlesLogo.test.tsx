import { render } from '@/src/tests/test-utils'
import { ParticlesLogo } from './ParticlesLogo'

describe('ParticlesLogo', () => {
  it('should render default markup', () => {
    const container = render(<ParticlesLogo />)

    expect(container).toMatchSnapshot()
  })
})
