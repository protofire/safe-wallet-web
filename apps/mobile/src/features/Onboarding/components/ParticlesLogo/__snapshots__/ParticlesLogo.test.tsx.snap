// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParticlesLogo should render default markup 1`] = `
<View>
  <RNSVGSvgView
    align="xMidYMid"
    bbHeight="180"
    bbWidth="181"
    fill="none"
    focusable={false}
    height="180"
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      [
        {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        {
          "flex": 0,
          "height": 180,
          "width": 181,
        },
      ]
    }
    vbHeight={180}
    vbWidth={181}
    width="181"
  >
    <RNSVGGroup
      fill={null}
      propList={
        [
          "fill",
        ]
      }
    >
      <RNSVGPath
        d="M33.8342 46.3745H24.0899C22.8299 46.3745 21.8084 47.396 21.8084 48.656V58.4003C21.8084 59.6604 22.8299 60.6818 24.0899 60.6818H33.8342C35.0942 60.6818 36.1157 59.6604 36.1157 58.4003V48.656C36.1157 47.396 35.0942 46.3745 33.8342 46.3745Z"
        fill={
          {
            "payload": 4279435136,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      />
      <RNSVGPath
        d="M40.0865 84.7895C41.288 84.7895 42.4895 84.7895 43.6919 84.7895C44.9519 84.7895 45.9734 83.768 45.9734 82.508V65.3846C45.9734 64.1246 44.9519 63.1031 43.6919 63.1031H26.5685C25.3085 63.1031 24.287 64.1246 24.287 65.3846V94.2242C24.287 95.4842 25.3085 96.5057 26.5685 96.5057H35.5244C36.7844 96.5057 37.8059 95.4842 37.8059 94.2242V87.071C37.8059 85.811 38.8274 84.7895 40.0874 84.7895H40.0865Z"
        fill={
          {
            "payload": 4279435136,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      />
      <RNSVGPath
        d="M112.749 141.736H101.54C99.8136 141.736 98.4138 143.136 98.4138 144.863V156.072C98.4138 157.798 99.8136 159.198 101.54 159.198H112.749C114.476 159.198 115.876 157.798 115.876 156.072V144.863C115.876 143.136 114.476 141.736 112.749 141.736Z"
        fill={
          {
            "payload": 4279435136,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      />
      <RNSVGPath
        d="M129.479 130.698H120.298C118.571 130.698 117.172 132.097 117.172 133.824V143.005C117.172 144.732 118.571 146.132 120.298 146.132H129.479C131.206 146.132 132.606 144.732 132.606 143.005V133.824C132.606 132.097 131.206 130.698 129.479 130.698Z"
        fill={
          {
            "payload": 4279435136,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      />
      <RNSVGPath
        d="M145.447 128.725H139.562C137.835 128.725 136.435 130.125 136.435 131.852V137.737C136.435 139.464 137.835 140.863 139.562 140.863H145.447C147.174 140.863 148.573 139.464 148.573 137.737V131.852C148.573 130.125 147.174 128.725 145.447 128.725Z"
        fill={
          {
            "payload": 4279435136,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      />
      <RNSVGPath
        d="M82.6709 140.864H70.7999C70.0925 140.864 69.5183 141.437 69.5183 142.145V154.016C69.5183 154.724 70.0916 155.298 70.7999 155.298H82.6709C83.3783 155.298 83.9525 154.725 83.9525 154.016V152.932C83.9525 152.224 84.5258 151.65 85.2341 151.65H89.9231C90.6305 151.65 91.2047 151.077 91.2047 150.369V144.708C91.2047 144 90.6314 143.426 89.9231 143.426H85.2341C84.5267 143.426 83.9525 142.853 83.9525 142.144C83.9525 141.437 83.3792 140.863 82.6709 140.863V140.864Z"
        fill={
          {
            "payload": 4279435136,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      />
      <RNSVGPath
        d="M51.3807 116.276H49.9722C48.6654 116.276 47.6061 115.217 47.6061 113.91V109.404C47.6061 108.097 46.5468 107.038 45.24 107.038H36.7341C35.4273 107.038 34.368 108.097 34.368 109.404V113.91C34.368 115.217 33.3087 116.276 32.0019 116.276H30.9876C29.6808 116.276 28.6215 117.336 28.6215 118.642V139.033C28.6215 140.34 29.6808 141.399 30.9876 141.399H45.2382C46.545 141.399 47.6043 142.458 47.6043 143.765C47.6043 145.072 48.6636 146.131 49.9704 146.131H60.6723C61.9791 146.131 63.0384 145.072 63.0384 143.765V133.063C63.0384 131.756 61.9791 130.697 60.6723 130.697H56.1102C54.8034 130.697 53.7441 129.638 53.7441 128.331V118.642C53.7441 117.336 52.6848 116.276 51.378 116.276H51.3807Z"
        fill={
          {
            "payload": 4279435136,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      />
      <RNSVGPath
        d="M159.192 86.1976V59.9491C159.192 58.9222 158.359 58.0906 157.333 58.0906H143.477C142.45 58.0906 141.618 57.2581 141.618 56.2321V47.8396C141.618 46.8127 140.786 45.9811 139.76 45.9811H136.099C135.072 45.9811 134.24 45.1486 134.24 44.1226V33.139C134.24 32.1121 133.408 31.2805 132.382 31.2805H118.863C117.836 31.2805 117.004 30.448 117.004 29.422V28.2394C117.004 27.2125 116.172 26.3809 115.146 26.3809H102.247C101.22 26.3809 100.388 25.5484 100.388 24.5224V22.6639C100.388 21.637 99.5558 20.8054 98.5298 20.8054H79.0403C78.0134 20.8054 77.1818 21.6379 77.1818 22.6639V42.0967C77.1818 43.1236 78.0143 43.9552 79.0403 43.9552H92.3333C93.3602 43.9552 94.1918 44.7877 94.1918 45.8137V47.3347C94.1918 48.3616 95.0243 49.1932 96.0503 49.1932H109.513C110.539 49.1932 111.371 50.0257 111.371 51.0517V52.291C111.371 53.3179 112.204 54.1495 113.23 54.1495H123.594C124.621 54.1495 125.452 54.982 125.452 56.008V60.2893C125.452 61.3162 126.285 62.1478 127.311 62.1478H129.029C130.056 62.1478 130.888 62.9803 130.888 64.0063V86.1994C130.888 87.2263 131.72 88.0579 132.746 88.0579H140.998C142.025 88.0579 142.857 88.8904 142.857 89.9164V92.2825C142.857 93.3094 142.024 94.141 140.998 94.141H138.858C137.831 94.141 136.999 94.9735 136.999 95.9995V107.209C136.999 108.236 136.167 109.067 135.141 109.067C134.114 109.067 133.282 109.9 133.282 110.926V123.149C133.282 124.176 134.115 125.007 135.141 125.007H147.364C148.391 125.007 149.222 124.175 149.222 123.149V117.234C149.222 116.207 150.055 115.376 151.081 115.376H156.376C157.402 115.376 158.234 114.543 158.234 113.517V95.9995C158.234 94.9726 157.402 94.141 156.376 94.141H155.418C154.391 94.141 153.559 93.3085 153.559 92.2825V89.9164C153.559 88.8895 154.392 88.0579 155.418 88.0579H157.333C158.36 88.0579 159.192 87.2254 159.192 86.1994V86.1976Z"
        fill={
          {
            "payload": 4279435136,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      />
      <RNSVGPath
        d="M62.6459 31.2787H55.7457C54.4857 31.2787 53.4641 32.3002 53.4641 33.5602C53.4641 34.8202 52.4426 35.8417 51.1826 35.8417H41.5508C40.2908 35.8417 39.2693 36.8632 39.2693 38.1232V55.9792C39.2693 57.2392 40.2908 58.2607 41.5508 58.2607H59.4069C60.6668 58.2607 61.6883 57.2392 61.6883 55.9792V51.9238C61.6883 50.6638 62.7098 49.6423 63.9698 49.6423H69.5462C70.8062 49.6423 71.8278 48.6208 71.8278 47.3608V33.5602C71.8278 32.3002 70.8062 31.2787 69.5462 31.2787H62.6459Z"
        fill={
          {
            "payload": 4279435136,
            "type": 0,
          }
        }
        propList={
          [
            "fill",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
